terraform/aws-public/mesos-slaves.tf

```diff
@@ -8,7 +8,7 @@ resource "atlas_artifact" "mesos-slave" {
 /* Mesos slave instances */
```

Comment:
Can we add a comment here exaplaining why is this being harcoded and linking to terraform issue?